src/PeerReview/Types.hs

```diff
@@ -53,11 +57,11 @@ data ReviewStatus = Waiting
                   | Accepted
                   deriving (Show)
 
-data ReviewTask = ReviewTask
-    { rtContent :: Text
-    , rtTaskId  :: TaskID
-    , rtUserId  :: UserID
-    } deriving (Show)
+data Task = Task
+    { tId      :: TaskID
```

Comment:
@keveri I'd add one field here, usually a Task would have:

- task unique identifier (sha)
- task name ("exConcat")
- task user (email)
- task content (actual text-content)

What do you think?